Resources/Maps/_RMC14/solaris.yml

```diff
@@ -22074,7 +22074,7 @@ entities:
     - type: Transform
       pos: 52.5,-112.5
       parent: 1
-- proto: CMSheetGlass25
```

Comment:
Is this change intentional?
You should be able to keep the 25 stack

Reply:
The prototypes are 10 30 and 50. I changed it to be like the metal sheets but didn't think too much about if the old prototypes were used in maps

Reply:
13 has some weird numbers sometimes, best to return the 25 stack and the one on Solaris